src/nitclk/examples/nitclk_master_trigger.py

```diff
@@ -0,0 +1,51 @@
+import argparse
+import niscope
+import nitclk
+import sys
+
+
+def example(resource_name1, resource_name2, channels, options, length, voltage):
+    with niscope.Session(resource_name=resource_name1, options=options) as session1, niscope.Session(resource_name=resource_name2, options=options) as session2:
+        session_list = [session1, session2]
+        for session in session_list:
+            session.configure_vertical(range=voltage, coupling=niscope.VerticalCoupling.DC)
+            session.configure_horizontal_timing(min_sample_rate=50000000, min_num_pts=length, ref_position=50.0, num_records=1, enforce_realtime=True)
+        session1.trigger_type = niscope.TriggerType.SOFTWARE
```

Comment:
I think you can use NI-TClk without triggering , just with trigger NONE. That's better than using a software trigger only to send it right away.

Reply:
@marcoskirsch I feel like a better example would be to still use a software trigger but insert some wait time between initiate and the sending of the trigger. That way the user can see the waveforms start at the same time even if the triggering is not immediate. If I just use the NONE trigger, that doesn't really show how to set up a real trigger.

Reply:
Ok, I suppose that depends on the goal of the examples. Do we want to teach customers how to synchronize instruments using `nitclk`, or do we want to teach them how to synchronize instruments using `nitclk` and get them started using a trigger?

I lean towards keeping it simple, `nimi-python` examples are for better or for worse supposed to help you to get started with the Python bindings but aren't meant to teach you how to use the APIs in depth.

The name of the example does say "trigger".

My preference is simplify and rename example but I leave it up to you.